### PR TITLE
application dev guides outline

### PR DESCRIPTION
**Pull request by flossypurse**
**PREVIEW**: https://deploy-preview-986--mystifying-fermi-1bc096.netlify.app/docs/operations/temporal-application-development

### CODE REVIEW

**Review by djmagee — COMMENTED** [review]
As we build out this section, it's gonna make a lot of people happy.

File: docs/operation/temporal-application-analysis.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - developer-guide
+---
+
+## Workflow Execution Visibility
+
+**Core concepts: [Standard Visibility](/docs/concepts/what-is-standard-visibility), [Advanced Visibility](/docs/concepts/what-is-advanced-visibility), [List Filter](/docs/concepts/what-is-a-list-filter), [Search Attribute](/docs/concepts/what-is-a-search-attribute)**
+
+The Temporal Platform h Visibility features that allow you to list Workflow Executions.
+
+- [How to list Workflow Executions using tctl](/docs/tctl/workflows/list)
```
Comment (flossypurse) [inline docs/operation/temporal-application-analysis.md:16]:
```suggestion
- [How to list Workflow Executions using tctl](/docs/tctl/workflow/list)
```

File: docs/operation/temporal-application-features.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- [What is a Side Effect?](/docs/concepts/what-is-a-side-effect)
+
+- [How to execute a Side Effect in Go](/docs/go/how-to-execute-a-side-effect-in-go)
+
+## Local Activities
+
+- [What is a Local Activity Execution](/docs/concepts/what-is-a-local-activity)
+
+- [How to set `ExecuteLocalActivityOptions` in Go](/docs/go/how-to-set-executelocalactivityoptions-in-go)
```
Comment (flossypurse) [inline docs/operation/temporal-application-features.md:42]:
```suggestion
- [How to set ExecuteLocalActivityOptions in Go](/docs/go/how-to-set-executelocalactivityoptions-in-go)
```

File: docs/operation/temporal-application-features.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+- [How to set a Heartbeat Timeout in Go](/docs/go/how-to-set-activityoptions-in-go/#heartbeattimeout)
+
+## Spawn Child Workflow Executions
+
+- [What is a Child Workflow Execution?](/docs/concepts/what-is-a-child-workflow-execution)
+
+- [How to spawn a Child Workflow Execution in Go](/docs/go/how-to-spawn-a-child-workflow-execution-in-go)
+
+- [How to set `ChildWorkflowOptions` in Go](/docs/go/how-to-set-childworkflowoptions-in-go)
```
Comment (flossypurse) [inline docs/operation/temporal-application-features.md:58]:
```suggestion
- [How to set ChildWorkflowOptions in Go](/docs/go/how-to-set-childworkflowoptions-in-go)
```

File: docs/operation/temporal-application-foundations.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+- [How to develop a Worker Program in Go](/docs/go/how-to-develop-a-worker-program-in-go)
+- [How to develop a Worker Program in Java](/docs/java/how-to-develop-a-worker-program-in-java)
+- [How to develop a Worker Program in PHP](/docs/php/how-to-develop-a-worker-program-in-php)
+- [How to develop a Worker Program in TypeScript](/docs/typescript/how-to-develop-a-worker-program-in-typescript)
+
+Each Worker Entity is created with the use of a Temporal Client.
+There are many customizations that can be set on the Client.
+
+- [How to set `ClientOptions` in Go](/docs/go/how-to-set-clientoptions-in-go)
```
Comment (flossypurse) [inline docs/operation/temporal-application-foundations.md:88]:
```suggestion
- [How to set ClientOptions in Go](/docs/go/how-to-set-clientoptions-in-go)
```

File: docs/operation/temporal-application-foundations.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+There are some customizations that can be set when registering Workflow Types and Activity Types.
+
+- [How to set `RegisterOptions` for Workflows in Go](/docs/go/how-to-set-registerworkflowoptions-in-go)
```
Comment (flossypurse) [inline docs/operation/temporal-application-foundations.md]:
```suggestion
- [How to set RegisterOptions for Workflows in Go](/docs/go/how-to-set-registerworkflowoptions-in-go)
```

File: docs/operations/application-development.md [hunk 6]
```diff
@@ -0,0 +1,862 @@
...
+
+:::
+
+This guide is meant to be a comprehensive resource for developing a [Temporal Application](/docs/concepts/what-is-a-temporal-application).
+
+It is broken down into four large sections:
+
+- [Foundations](#application-foundations): The minimum things required to build and run a simple Workflow with a single Activity.
+- [Features](#application-features): All of the general features available to a Temporal Application.
+- [Analysis](#application-analysis): Features and tools available that give you insight into you application.
```
Comment (rachfop) [inline docs/operations/application-development.md:26]:
```suggestion
- [Analysis](#application-analysis): Features and tools available that give you insight into your application.
```

**Comment by flossypurse** [comment]
This was a POC.

Closing in favor of other incremental PRs.

**Comment by flossypurse** [comment]
This was a POC.

Closing in favor of other incremental PRs.

**Pull request closed without merging**